lib/db/mysql/schema.sql

```diff
@@ -31,7 +31,8 @@ CREATE TABLE IF NOT EXISTS codes (
   scope VARCHAR(256) NOT NULL,
   createdAt TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
   authAt BIGINT DEFAULT 0,
-  offline BOOLEAN DEFAULT FALSE
+  offline BOOLEAN DEFAULT FALSE,
+  profileChangedAt BIGINT DEFAULT 0
```

Comment:
I wonder whether `0` or `NULL` would be a better default here.  It probably won't matter either way, but defaulting to `NULL` in the db and them omitting `profile_changed_at` for tokens where we don't know it may be a little more explicit.